css-escape/index.html

```diff
@@ -0,0 +1,41 @@
+---
+feature_name: CSS.escape()
+chrome_version: 46
+feature_id: 6526863593701376
+---
+
+<h3>Background</h3>
+<p>
+  There are certain strings that are valid as when used as an HTML element's <code>id</code> or
+  <code>class</code> attributes, but not valid as a CSS selector string. They include strings
+  that begin with numbers, contain spaces, or contain characters with special meaning in CSS.
+  Mathias Bynens's <a href="https://mathiasbynens.be/notes/css-escapes">explanation</a> delves into
+  the full details.
+</p>
+<p>
+  The <code>CSS.escape()</code> method provides a convenient way to escape a string so that conforms
+  to the CSS selector requirements.
```

Comment:
Maybe add something like this here:

> This is useful when dynamically building a CSS selector based on (untrusted) user input.

[Some examples of that from my CSS Day 2014 talk:](https://vimeo.com/100264064#t=20:25)
* `document.querySelector(location.hash)`
* `document.querySelector('a[href="' + someValue + '"]')`